lambdas/es/indexer/test/test_index.py

```diff
@@ -268,3 +268,29 @@ def test_gzipped_notebook_contents(self):
 
         contents = index.get_contents('test-bucket', 'foo.ipynb.gz', '.gz', etag='etag', version_id=None, s3_client=self.s3_client, size=123)
         assert "Model results visualization" in contents
+
+    def test_parquet_contents(self):
+        parquet = (BASE_DIR / 'amazon-reviews-1000.snappy.parquet').read_bytes()
+
+        self.s3_stubber.add_response(
+            method='get_object',
+            service_response={
+                'Metadata': {},
+                'ContentLength': 123,
+                'Body': BytesIO(parquet),
+            },
+            expected_params={
+                'Bucket': 'test-bucket',
+                'Key': 'foo.parquet',
+                'IfMatch': 'etag',
+            }
+        )
+
+        contents = index.get_contents('test-bucket', 'foo.parquet', '.parquet', etag='etag', version_id=None, s3_client=self.s3_client, size=123)
+        size = len(contents.encode('utf-8', 'ignore'))
+        # we know the file is bigger than the limit
+        assert size == index.DOC_LIMIT_BYTES
+        # spot check for contents
+        assert "This is a great handy reference tool for all" in contents
+        assert "I recieved a Marvel the Mustang for Christmas" in contents
```

Comment:
Ugh... "recieved". But that's from the original file?

Reply:
yeah this is the original file